tests/roxctl/bats-tests/local/deployment-bundles-psps.bats

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bats
+
+load "../helpers.bash"
+
+out_dir=""
+
+setup_file() {
+    # remove binaries from the previous runs
+    [[ -n "$NO_BATS_ROXCTL_REBUILD" ]] || rm -f "${tmp_roxctl}"/roxctl*
+    echo "Testing roxctl version: '$(roxctl-release version)'" >&3
+
+    command -v grep || skip "Command 'grep' required."
+    [[ -n "${API_ENDPOINT}" ]] || fail "Environment variable 'API_ENDPOINT' required"
+    [[ -n "${ROX_PASSWORD}" ]] || fail "Environment variable 'ROX_PASSWORD' required"
```

Comment:
Since these tests require a running cluster, they should not go to `bats-test/local` directory, rather should be in `bats-tests/cluster`.